tests/hw8/test_task2.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from hw8.task2 import TableData
+
+
+@pytest.fixture(scope="module")
+def table_data_instance():
```

Comment:
the same as for test_task1 - this would be executed multiple times

Reply:
Done